worker-document/src/main/java/com/hpe/caf/worker/document/BulkDocumentWorkerAdapter.java

```diff
@@ -44,5 +44,6 @@ public void processTasks(final BulkWorkerRuntime runtime) throws InterruptedExce
             = new BulkDocumentMessageProcessor(application, bulkDocumentWorker, runtime);
 
         messageProcessor.processTasks();
+        messageProcessor.closeBindings();
```

Comment:
This module didn't open the bindings so why would it be responsible for closing them.